electron/execution.ts

```diff
@@ -103,7 +107,9 @@ async function recordJourneys(data, browserWindow) {
     const closeBrowser = async () => {
       browserContext = null;
       actionListener.removeListener('actions', actionsHandler);
-      await browser.close().catch({});
+      await browser.close().catch(err => {
```

Comment:
Why don't we go ahead and use this syntax instead of combining `async...await` with callbacks.

```suggestion
      try {
        await browser.close();
      } catch (e) {
        logger.error("Browser close threw an error", e);
      }
```

Reply:
agreed 👍